pkg/administration/events/queue.go

```diff
@@ -0,0 +1,66 @@
+package events
+
+import (
+	"container/list"
+
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+const (
+	maxQueueSize = 100000
```

Comment:
do we have any back of envelope math on size of each message so if we hit 100k we think we'd be okay memory wise?

Reply:
Added a comment, 100k events should be around 16MB in memory.